<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmemberdialog</class>
 <widget class="QDialog" name="addmemberdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QComboBox" name="cbCamList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCapture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInitCam">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Init Camera</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>333</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbFirstname">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="lFirstname">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLastname">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lastname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbLastname">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QLabel" name="lAge">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Age (year)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lHeight">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (cm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lWeight">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>250</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight (kilograms)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSport">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>310</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sports</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbSport">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QLabel" name="lIllness">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Illnesses</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbIllness">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>400</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
  </widget>
  <widget class="QLabel" name="lRFID">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RFID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbRFID">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>460</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse Picture</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAge">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>150</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbWeight">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbHeight">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>560</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbYear">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMonth">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDay">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lExpire">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expire Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSex">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSex">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
